gammapy/estimators/points/tests/test_core.py

```diff
@@ -336,3 +337,22 @@ def test_flux_points_plot_no_error_bar():
     flux_points = FluxPoints.from_table(table)
     with mpl_plot_check():
         _ = flux_points.plot(sed_type="dnde")
+
+
+def test_fp_no_is_ul():
+    dataset_1 = simulate_map_dataset(random_state=0, name="dataset_1")
```

Comment:
It might be better not to import and use here the estimator. I would rather suggest to import a `FluxPoints` from a table where you remove the UL column.
See e.g. `test_plot_fp_no_ul`

Reply:
I utilised the `simulate_map_dataset`, so that the `FluxPoints` has the `has_ul` set to False.
By importing the flux points from `GAMMAPY_DATA` mean `has_ul` is set to True. 
Open to suggestion on if you still wish to import the `FluxPoints` from a table

Reply:
Even if you remove the `ul` column from the input table before creating the `FluxPoints` object?

Reply:
Apologies, it works now. Thanks @registerrier